docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/SelectColumns.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class SelectColumns
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = mlContext.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Age    Case  Education  induced     parity  pooled.stratum  row_num  ...
+            // 26.0   1.0   0-5yrs      1.0         6.0       3.0      1.0  ...
+            // 42.0   1.0   0-5yrs      1.0         1.0       1.0      2.0  ...
+            // 39.0   1.0   0-5yrs      2.0         6.0       4.0      3.0  ...
+            // 34.0   1.0   0-5yrs      2.0         4.0       2.0      4.0  ...
+            // 35.0   1.0   6-11yrs     1.0         3.0       32.0     5.0  ...
+
+            // Select a subset of columns to keep, but don't drop the others.
+            var pipeline = mlContext.Transforms.SelectColumns(new string[] { "Age", "Education" });
+
+            // Now we can transform the data and look at the output to confirm the behavior of CopyColumns.
+            // Don't forget that this operation doesn't actually evaluate data until we read the data below.
+            var transformedData = pipeline.Fit(trainData).Transform(trainData);
+
+            // Print the number of columns schema
```

Comment:
>columns schema [](start = 35, length = 14)

Print the number of columns in the schema.

Reply:
Done.